Header header
string label
int32 x_center
int32 y_center
int32 w
int32 h
float32 confidence
int32 xmin
int32 xmax
int32 ymin
int32 ymax
int32 size
int32 id
